clang/lib/CIR/CodeGen/CIRGenExprAgg.cpp

```diff
@@ -29,6 +29,73 @@ using namespace cir;
 using namespace clang;
 
 namespace {
+
+/// Is the value of the given expression possibly a reference to or
+/// into a __block variable?
+static bool isBlockVarRef(const Expr *E) {
```

Comment:
`isBlockVarRed` is just  a copy-pasta from `CGExprAgg.cpp`

Reply:
I usually leave a comment on those, something along the lines of `// FIXME(cir): This should be a common helper between CIRGen and traditional CodeGen", can you add that to this helper?

Reply:
@bcardosolopes done!